packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1218,15 +1223,31 @@ export default mixins(
 				// @ts-ignore
 				this.instance.setZoom(zoomLevel);
 			},
+			async setRecenteredCanvasAddButtonPosition () {
+				// We need to wait two Vue component tick for the CanvasAddButton to render
+				// so we could grab its dimensions
+				await this.$nextTick();
+				await this.$nextTick();
```

Comment:
could we add an event when button get mounted instead? less likely to cause issues in the future